### ISSUE TITLE
Document / point to `path_search` rippled.cfg options

### ISSUE THREAD

**Issue by ximinez** [XRPLF/xrpl-dev-portal#1388]
Per https://github.com/ripple/rippled/issues/4141, rippled recently reduced the default values of the `path_search`-related configuration options to lower memory consumption. It might be helpful to document and/or point out these options on the pages that document path finding and/or server config for operators who desire the old behavior, which was able to find longer paths, even though it consumed more memory.

**Comment by mDuo13** [comment]
I agree it would be a good idea to add a new page or tutorial clarifying how & why a server operator may want to increase or decrease the path search setting on their server.
